saleor/graphql/account/mutations.py

```diff
@@ -12,22 +31,39 @@ class UserInput(graphene.InputObjectType):
     note = graphene.String(description='A note about the user.')
 
 
+class UserCreateInput(UserInput):
+    send_password_email = graphene.Boolean(
+        description='Send an email with a link to set a password')
+
+
 class StaffInput(UserInput):
     permissions = graphene.List(
         graphene.String,
         description='List of permission code names to assign to this user.')
 
 
+class StaffCreateInput(StaffInput):
+    send_password_email = graphene.Boolean(
+        description='Send an email with a link to set a password')
+
+
 class CustomerCreate(ModelMutation):
     class Arguments:
-        input = UserInput(
+        input = UserCreateInput(
             description='Fields required to create a customer.', required=True)
 
     class Meta:
         description = 'Creates a new customer.'
         exclude = ['password']
         model = models.User
 
+    @classmethod
+    def save(cls, info, user, cleaned_input):
+        user.save()
+        if cleaned_input.get('send_password_email'):
+            send_user_password_reset_email(user)
+            return cls.success_response(user)
```

Comment:
This line can be dropped, as `ModelMutation` doesn't expect any return value from `save`. It is the `mutate` method that handles returning GraphQL responses.